### PR TITLE
:bug: Fix Task config property

### PR DESCRIPTION
**Pull request by rubenaeg**
## Proposed changes
This PR fixes a small bug where `indent` was used in Task initialization instead of the correct `indentation`.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [x] My code follows the code style of this project
- [ ] My change requires a change to the documentation
- [ ] I have updated the documentation accordingly
- [ ] I have added tests to cover my changes
- [x] All new and existing tests passed

### CODE REVIEW

**Review by jankoenig — APPROVED** [review]

File: integrations/plugin-debugger/src/JovoDebugger.ts [hunk 1]
```diff
@@ -464,15 +464,6 @@ export class JovoDebugger extends Plugin<JovoDebuggerConfig> {
     await this.emitDebuggerConfig();
     await this.emitLanguageModelIfEnabled();
 
-    // eslint-disable-next-line @typescript-eslint/no-unused-vars
-    function propagateStreamAsLog(stream: Writable, socket: typeof Socket) {
```
Comment (m-ripper) [inline integrations/plugin-debugger/src/JovoDebugger.ts:469]: Please add this function again. It is not used now but might be used in the future.

**Review by m-ripper — CHANGES_REQUESTED** [review]
Beside that one change in the Debugger, it looks good. 
Did you test the unknown typing? Due to unknown being stricter further error could occurre in integrations or apps.

File: integrations/plugin-debugger/src/JovoDebugger.ts (hunk 1 above)
Comment (rubenaeg) [reply integrations/plugin-debugger/src/JovoDebugger.ts:469]: Added it in `utilities.ts` :+1:

Comment (m-ripper) [reply integrations/plugin-debugger/src/JovoDebugger.ts:469]: Can you remove the `@ts-ignore` please? Casting would be a much cleaner solution if necessary.

**Pull request merged**